cdap-common/src/main/java/co/cask/cdap/common/security/AuditEnum.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.common.security;
+
+/**
+ * Declares which content needs to be audit logged.
+ */
+public enum AuditEnum {
```

Comment:
Better not call this AuditEnum. We don't generally end the name of an enum type in Enum. Maybe AuditDetail, or if you know something better.